diracx-core/src/diracx/core/settings.py

```diff
@@ -102,3 +113,75 @@ class DevelopmentSettings(ServiceSettingsBase):
     # When then to true (only for demo/CI), crash if an access policy isn't
     # called
     crash_on_missed_access_policy: bool = False
+
+    @classmethod
+    def create(cls) -> Self:
+        return cls()
+
+
+class AuthSettings(ServiceSettingsBase):
+    """Settings for the authentication service."""
+
+    model_config = SettingsConfigDict(env_prefix="DIRACX_SERVICE_AUTH_")
+
+    dirac_client_id: str = "myDIRACClientID"
+    # TODO: This should be taken dynamically
+    # ["http://pclhcb211:8000/docs/oauth2-redirect"]
+    allowed_redirects: list[str] = []
+    device_flow_expiration_seconds: int = 600
+    authorization_flow_expiration_seconds: int = 300
+
+    # State key is used to encrypt/decrypt the state dict passed to the IAM
+    state_key: FernetKey
+
+    # TODO: this should probably be something mandatory
+    # to set by the user
+    token_issuer: str = "http://lhcbdirac.cern.ch/"  # noqa: S105
```

Comment:
Is this something used (e.g. by the tests) or something here as an example?

Reply:
I don't think it's used, let's try to remove it and see if it breaks